docs/user-event/options.mdx

```diff
@@ -8,8 +8,19 @@ can be applied per [`setup()`](setup.mdx).
 
 ### advanceTimers
 
-If you are using fake timers, you need to advance your timers when we internally
-[delay](#delay) subsequent code.
+`user-event` adds a [delay](#delay) between some subsequent inputs. When using
```

Comment:
I think we're missing the exact example we had in the commit before because currently, it doesn't make a lot of sense. I'd remove all of the examples and only keep what we had before :)
```js
const user = userEvent.setup({advanceTimers: jest.advanceTimersByTime})
```

Reply:
I agree

Reply:
This looks much better to me, thanks.
I'll wait for @timdeschryver to respond as he also reviewed this one :)